core/run/src/lib.rs

```diff
@@ -586,14 +578,35 @@ impl Axon {
         Ok(())
     }
 
+    fn run_jaeger(config: Config) {
+        if let Some(jaeger_config) = config.jaeger {
+            let service_name = match jaeger_config.service_name {
+                Some(name) => name,
+                None => "axon".to_string(),
+            };
+
+            let tracing_address = match jaeger_config.tracing_address {
+                Some(address) => address,
+                None => std::net::SocketAddr::from(([0, 0, 0, 0], 6831)),
+            };
+
+            let tracing_batch_size = jaeger_config.tracing_batch_size.unwrap_or(50);
+
+            global_tracer_register(&service_name, tracing_address, tracing_batch_size);
+            log::info!("jaeger start");
+        };
+    }
+
     fn run_prometheus_server(config: Config) {
-        let prometheus_listening_address = match config.apm {
-            Some(apm_config) => apm_config.prometheus_listening_address,
+        let prometheus_listening_address = match config.prometheus {
+            Some(prometheus_config) => prometheus_config.listening_address.unwrap(),
             None => std::net::SocketAddr::from(([0, 0, 0, 0], 8100)),
         };
         tokio::spawn(async move {
```

Comment:
just `tokio::spawn(run_prometheus_server(prometheus_listening_address))` is enough

Reply:
> run_prometheus_server(prometheus_listening_address)

OK, I've changed it on my end